lib/Cron/AutoExport.php

```diff
@@ -9,24 +9,35 @@
 
 namespace OCA\Cospend\Cron;
 
-use OC\BackgroundJob\TimedJob;
 use OCA\Cospend\Service\ProjectService;
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\BackgroundJob\TimedJob;
 
 class AutoExport extends TimedJob {
-
 	/**
 	 * @var ProjectService
 	 */
 	private $projectService;
 
-	public function __construct(ProjectService $projectService) {
+	/**
+	 * @param ITimeFactory $time
+	 * @param ProjectService $projectService
+	 */
+	public function __construct(ITimeFactory $time, ProjectService $projectService)
+	{
```

Comment:
Could you keep the function opening curly bracket on the same line?

```suggestion
	public function __construct(ITimeFactory $time, ProjectService $projectService) {
```

Reply:
I changed it and also changed the code style settings in my ide to automatically use the "same line" option.